migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -68,25 +66,52 @@ public static void main(String[] args) throws IOException {
     } else {
       MigrationJob migrationJob;
       if (clm.numberInput.number != null) {
-        migrationJob = MigrationFactory.createMigration(clm.numberInput.number, clm.regionCode);
+        migrationJob = MigrationFactory.createMigration(clm.numberInput.number, clm.countryCode);
       } else {
         migrationJob = MigrationFactory
-            .createMigration(Paths.get(clm.numberInput.file), clm.regionCode);
+            .createMigration(Paths.get(clm.numberInput.file), clm.countryCode);
       }
 
-      System.out.println(migrationJob.getRecipesCsvTable());
-      System.out.println(migrationJob.getMigrationEntries().stream()
-          .map(MigrationEntry::getOriginalNumber)
-          .collect(Collectors.toList()));
+      MigrationReport mr =  migrationJob.getMigrationReportForCountry();
+      if (clm.numberInput.file != null) {
+        printFileReport(mr, Paths.get(clm.numberInput.file));
+      } else {
+        printNumberReport(mr);
+      }
+    }
+  }
+
+  private static void printFileReport(MigrationReport mr, Path originalFile) throws IOException {
+    String newFile = mr.exportToFile(originalFile.getFileName().toString());
+    Scanner scanner = new Scanner((System.in));
+    String response = "";
 
-      DigitSequence demoRecipeOldPrefix = DigitSequence.of("84120");
-      int demoRecipeOldLength = 12;
-      RangeKey key = RangeKey.create(RangeSpecification.from(demoRecipeOldPrefix),
-          Collections.singleton(demoRecipeOldLength));
-      System.out.println("\nAll migrations for key " + key + ":");
-      migrationJob.getMigratedNumbersForRecipe(key).forEach(res -> System.out.println("\t" + res));
-      System.out.println("\nAll migrations for region " + migrationJob.getRegionCode() + ":");
-      migrationJob.getMigratedNumbersForRegion().forEach(res -> System.out.println("\t" + res));
+    System.out.println("New numbers file created at: " + newFile);
+    while (!response.equals("0")) {
+      System.out.println("\n(0) Exit");
+      System.out.println("(1) Print Metrics");
+      System.out.print("Select from the above options: ");
+      response = scanner.nextLine();
+      if (response.equals("1")) {
+        mr.printMetrics();
+      }
+    }
+  }
+
+  private static void printNumberReport(MigrationReport mr) {
+    if (mr.getValidMigrations().size() == 1) {
+      System.out.println("Successful migration into: +"
+          + mr.getValidMigrations().get(0).getMigratedNumber());
+    } else if (mr.getInvalidMigrations().size() == 1) {
+      System.out.println("The number was migrated into '+"
+          + mr.getInvalidMigrations().get(0).getMigratedNumber() + "' but this number was not "
+          + "seen as being valid and dialable when inspecting our data for the given country");
```

Comment:
Could we mention the country they passed in for these print statements, just as a sanity check for the user.

Reply:
Country code print out added above in main method